src/server/qgsmsutils.h

```diff
@@ -20,6 +20,13 @@
 //! Some utility functions that may be included from everywhere in the code
 namespace QgsMSUtils
 {
+  struct QgsLayerCoordinateTransform
+  {
+    QString srcAuthId;
```

Comment:
Would be great if you could add some doxygen comments to highlight the meaning of these variables.